docs/learning-the-basics.md

```diff
@@ -6,75 +6,75 @@ As with every operating system there are some things that you may not be familia
 
 ### The Panel {#the-panel}
-At the top of the screen you can see the Panel. On the left is the Applications Menu, in the center are the time and date, and on the right are the Indicators.
+At the top of the screen you can see the panel. On the left is the application menu, in the center are the time and date, and on the right are the Indicators.
 
 ![The Panel](../images/docs/learning-the-basics/panel.png)
 
 #### Applications Menu {#applications}
 
-On the left side of the panel is the Applications Menu. Selecting **Applications** opens the menu with all of your installed apps. You can view multiple pages of apps using the pagers at the bottom or by scrolling. You can also use the view switcher at the top to switch between a grid view and a category view.
+On the left side of the panel is the application menu. Selecting **Applications** opens the menu with all of your installed apps. You can view multiple pages of apps using the pagers at the bottom or by scrolling. You can also use the view switcher at the top to switch between a grid view and a category view.
 
 ![Applications Menu](../images/docs/learning-the-basics/slingshot.png)
 
-You can search for apps by name or by keyword and perform actions associated with them. You can also search for System Settings panes. Some of the actions you can find in search include:
+You can search for apps by name or by keyword and perform actions associated with them. You can also search for system settings panes. Some of the actions you can find in search include:
 
 * Shutdown and Restart
 * Compose Message
 * New Private Browsing Window
-* Math operations (example: 23 * 5)
+* Math Operations (example: 23 * 5)
 
-To open the Applications Menu with the keyboard, press <kbd>⌘</kbd> + <kbd>space</kbd>.
+To open the application menu with the keyboard, press <kbd>⌘</kbd> + <kbd>space</kbd>.
 
 #### Indicators {#indicators}
-On the right side of the panel are icons called Indicators. These tell you of the current status of your session, i.e. your network connections, battery power, sound input and output, notifications, etc. Selecting an indicator exposes more information and related actions.
+On the right side of the panel are icons called indicators. These tell you of the current status of your session, i.e. your network connections, battery power, sound input and output, notifications, etc. Selecting an indicator exposes more information and related actions.
 
 A middle click or three finger tap on an indicator will present you with the following quick actions:
 
 * Sound: Toggle the mute state of the primary output device
-* Bluetooth: Toggle the Enabled/Disabled the primary bluetooth device
+* Bluetooth: Toggle on or off the primary Bluetooth device
```

Comment:
While we're here, I think it would be more concise and effectively as accurate to say, "Toggle on or off". Most devices are only going to have one Bluetooth radio, so specifying "the primary" feels overly precise